docs/proxy/datacenter_proxy.md

```diff
@@ -6,107 +6,89 @@ paths:
+## [](#ip-address-rotation) IP address rotation
 
-HTTP proxy username is used to pass various parameters for the proxy connection. For example, the username can look as follows:
+Web scrapers can rotate the IP addresses they use to access websites. They assign each request a different IP address, which makes it appear like they are all coming from different users.
 
-    groups-SHADER,session-rand123456
+Depending on whether you use a [browser](https://apify.com/apify/web-scraper) or [HTTP requests](https://apify.com/apify/cheerio-scraper) for your scraping jobs, IP address rotation works differently.
 
-The following table describes the available parameters:
+* Browser – a different IP address is used for each browser.
+* HTTP request – a different IP address is used for each request.
 
-<table class="table table-bordered table-condensed">
-    <tbody>
-    <tr>
-        <th><code>groups</code></th>
-        <td>
-            If specified, all proxied requests will use proxy servers from selected proxy groups.
-            <br/>For example <code>groups-SHADER+BUYPROXIES94952</code>.
-            <br /><strong>This parameter is optional</strong>,
-            by default, the proxy uses all available proxy servers
-            <br/>from all groups the user has access to.
-        </td>
-    </tr>
-    <tr>
-        <th><code>session</code></th>
-        <td>
-            If specified, all proxied requests with the same session identifier are routed
-            <br/>through the same IP address. For example <code>session-rand123456</code>.
-            <br /><strong>This parameter is optional</strong>, by default, each proxied request
-            is assigned a
-            <br/>randomly picked least used IP address.
-            <br /><strong>The session string can only contain numbers (0-9), letters (a-z or A-Z),
-            dot (.),
-            <br/>underscore (_), a tilde (~) and the maximum length is 50 characters!</strong>
-        </td>
-    </tr>
-    <tr>
-        <th><code>country</code></th>
-        <td>
-            If specified, all proxied requests will use proxy servers from a selected country.
-             <br/>Please be aware that if there are no proxy servers
-            <br/>from the specified country the connection will fail.
-             <br/>For example <code>groups-SHADER,country-US</code>.
-            <br /><strong>This parameter is optional</strong>,
-            by default, the proxy uses all available
-            <br/>proxy servers from all countries.
-        </td>
-    </tr>
-    </tbody>
-</table>
+[Click here]({{@link web_scraping_101/anti_scraping_techniques.md#bypassing-ip-address-based-blocking}}) to learn more about IP address rotation and other ways of bypassing blocking.
 
-If you do not want to specify both `groups` and `session` parameters and therefore use **default** behavior for both, use the following username:
+## [](#session-persistence) Session persistence
```

Comment:
What about changing this section from session persistence to `IP rotation` and rewriting it a bit? The idea is that a lot of people have no idea what a `session` is, whereas IP is a generally known term.

I remember, 3 months into my first engineering job, where I was building a Node.js server framework, I had no idea what a session was. 🤦  So I think it's fair to assume 99% of people are clueless about sessions.

Reply:
@mnmkng There is already an `IP rotation` section above this, do you mean I should join them together?

Or maybe there's a clearer way to explain this?

Also, there's a bit in `proxy.md` that explains sessions a little - I'll add a link to it.

Reply:
Yeah, sorry I did not realize the above section was named IP rotation. What I meant was:

There's no clear connection between "Session persistence" and "IP rotation". Under the IP rotation section, there is generic information about IP rotation and what it is, and in the Session persistence, we talk about persisting sessions. But there is very little "glue" that would tie those two together. I think the fact that's the most important: sessions represent IPs and you can therefore manage IPs by managing sessions, should have a more prominent place and should be explained very clearly.